lib/perseo.js

```diff
@@ -48,7 +48,10 @@ var domain = require('domain'),
     serviceMiddleware = require('./middleware/service'),
     errorMiddleware = require('./middleware/error'),
     myutils = require('./myutils'),
-    d = domain.create();
+    d = domain.create(),
+    swaggerJsdoc = require('swagger-jsdoc'),
+    swaggerUi = require('swagger-ui-express'),
+    pjson = require('./package.json');
```

Comment:
```suggestion
    pjson = require('../package.json');
```

Not sure, please check.

Reply:
it is correct

Reply:
Ok. Fixed in df81a41f6f55ee0023a29090136c24d0955ca20a